### PR TITLE
testing for messages.py

### PR DESCRIPTION
**Pull request by Yash-Vardhan-Sharma**
## Description
Added tests for message.py file

## Relevant Issue
#104

## Affected Dependencies
None

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/RecoEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/RecoEdge/blob/main/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/NimbleEdge/RecoEdge/labels)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by vkkhare** [comment]
You should fill the Issue template @Yash-Vardhan-Sharma for the PRs

File: test/test_messages.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+def test_job_type(job_type, job_args , job_kwargs,
+                  senderid, receiverid, workerstate):
+
+    obj = JobSubmitMessage(job_type, job_args, job_kwargs,
+                           senderid, receiverid, workerstate)
+
+    assert type(obj.get_job_type()) == str
+    assert obj.get_job_type() == job_type
+
+def test_jobresponse_status(job_type, senderid, receiverid):
```
Comment (vkkhare) [inline test/test_messages.py:29]: dont just check the type, check their value as well

File: test/test_messages.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    obj = JobResponseMessage(job_type, senderid, receiverid)
+
+    assert(type(obj.status) == bool)
+
+def test_jobsubmitmessage(job_type, job_args , job_kwargs,
+                          senderid, receiverid, workerstate):
+
+    obj = JobSubmitMessage(job_type, job_args, job_kwargs,
+                           senderid, receiverid, workerstate)
+
```
Comment (vkkhare) [inline test/test_messages.py:40]: check worker state as well

File: test/test_messages.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+with open("test_config.yml", 'r') as cfg:
+    config = yaml.load(cfg, Loader=yaml.FullLoader)
+
+
+def pytest_generate_tests(metafunc):
+    fct_name = metafunc.function.__name__
+    if fct_name in config:
+        params = config[fct_name]
+        metafunc.parametrize(params["params"], params["values"])
+
```
Comment (vkkhare) [inline test/test_messages.py:18]: run tests for Message class as well

**Review by vkkhare — CHANGES_REQUESTED** [review]

File: test/test_messages.py (hunk 3 above)
Comment (Yash-Vardhan-Sharma) [reply test/test_messages.py:18]: done

File: test/test_messages.py (hunk 2 above)
Comment (Yash-Vardhan-Sharma) [reply test/test_messages.py:65]: done

Comment (vkkhare) [reply test/test_messages.py:65]: assertion is only for class type?

Comment (vkkhare) [reply test/test_messages.py:65]: Create a new method to test workerstate.

Also rebase once

File: test/test_messages.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+def test_worker_state(job_type, job_args , job_kwargs,
+                  senderid, receiverid, workerstate):
+    """test get_worker_state method
+    """
+
+    obj = JobSubmitMessage(job_type, job_args, job_kwargs,
+                           senderid, receiverid, workerstate)
+    
+    assert obj.get_worker_state() == workerstate
```
Comment (Yash-Vardhan-Sharma) [inline test/test_messages.py:37]: @vkkhare this is the method to test workerstate output

**Comment by vkkhare** [comment]
Rebase and I will merge then

**Pull request closed without merging**